### ISSUE TITLE
New branch (non-refactored version) release date

### ISSUE THREAD

**Issue by 56luca** [imperial-qore/TranAD#25]
Hello,

you mention in multiple GitHub issues (#10 #18 #7 #11 ) that you intend to release another branch which includes the final models, loss functions, masks, and tuned hyperparameters. Some of these issues are now over 2 months old. Do you have a specific time schedule for the publication of this version? Is there any information you can give on when we will be able to access it?

Thank you and best regards!

**Comment by shreshthtuli** [comment]
Thanks for your interest in the project. Cross checking all hyperparameter values and results is a time consuming activity. We are planning to resolve all issues and also release the intended codebase before the conference presentation, i.e., by September. Stay tuned.

**Comment by eventhorizon7** [comment]
Hello, will the full version of the code be released soon?

**Comment by shreshthtuli** [comment]
We have added params in src/params.json.
